intera_examples/scripts/send_random_trajectory.py

```diff
@@ -0,0 +1,160 @@
+    parser.add_argument(
+        "-s",  "--speed_ratio", type=float, default=0.5,
+        help="A value between 0.0 (slow) and 1.0 (fast)")
+    parser.add_argument(
+        "-a",  "--accel_ratio", type=float, default=0.5,
+        help="A value between 0.0 (slow) and 1.0 (fast)")
+    parser.add_argument(
+        "-o", "--output_file",
+        help="Save the trajectory task to a bag file")
+    parser.add_argument(
+        "--output_file_type", default="yaml",
+        choices=["csv", "yaml"], help="Select the save file type")
+    parser.add_argument(
+        "-p", "--print_trajectory", action='store_true', default=False,
+        help="print the trajectory after loading")
+    parser.add_argument(
+        "--do_not_send", action='store_true', default=False,
+        help="generate the trajectory, but do not send to motion controller.")
+    parser.add_argument(
+        "--log_file_output",
+        help="Save motion controller log messages to this file name")
+    args = parser.parse_args()
+
+    if args.waypoint_count < 1:
+        args.waypoint_count = 1
+        rospy.logwarn('Input out of bounds! Setting waypoint_count = 1')
+
+    try:
+        rospy.init_node('send_random_joint_trajectory_py')
+
+        # Set the trajectory options
+        traj_opts = TrajectoryOptions()
+        traj_opts.interpolation_type = args.trajType
+        traj = MotionTrajectory(trajectory_options = traj_opts)
+
+        # Set the waypoint options
+        wpt_opts = MotionWaypointOptions(max_joint_speed_ratio=args.speed_ratio,
+                                        max_joint_accel=args.accel_ratio)
+        waypoint = MotionWaypoint(options = wpt_opts.to_msg())
+
+        # Append a waypoint at the current pose
+        limb = Limb()
+        waypoint.set_joint_angles(limb.joint_ordered_angles())
+        traj.append_waypoint(waypoint.to_msg())
+
+        # Set up the random walk generator
+        walk = RandomWalk()
+        limits = JointLimits()
+        walk.set_lower_limits(limits.get_joint_lower_limits(limb.joint_names()))
+        walk.set_upper_limits(limits.get_joint_upper_limits(limb.joint_names()))
+        walk.set_last_point(waypoint.get_joint_angles())
+        walk.set_boundary_padding(args.boundaryPadding)
+        walk.set_maximum_distance(args.stepDistance)
+
+        for i in range(0, args.waypoint_count):
+            joint_angles = walk.get_next_point()
+            waypoint.set_joint_angles(joint_angles = joint_angles)
+            traj.append_waypoint(waypoint.to_msg())
+
+        if args.output_file is not None:
+            if args.output_file_type == "csv":
+                traj.to_csv_file(args.output_file)
+            elif args.output_file_type == "yaml":
+                traj.to_yaml_file(args.output_file)
+            else:
+                rospy.logwarn("Did not recognize output file type")
+
+        if args.print_trajectory:
+            rospy.loginfo('\n' + traj.to_string())
```

Comment:
It's a bit unusual to have a newline before a log info message to print. What's the rational here?